README.md

```diff
@@ -26,7 +26,22 @@
           "app_category_type": "public.app-category.productivity"
         },
         "linux": {
-          "icon": "resources/linux/<your-project-id>.png"
+          "icon": "resources/linux/<your-project-id>.png",
+          "debian_config": {
+            "section": "Databases",
+            "depends": [
```

Comment:
@imlucas I'm now pretty sure that https://github.com/unindented/electron-installer-debian/blob/e42b311/src/installer.js#L191 acts as an explicit override which is what I'd expect given how `_.defaults` works:

```
> _.defaults({}, {x:[]}, {x:2})
{ x: [] }
```

Testing `DEBUG=* npm run release` also supports it being an explicit override:

![confirmed it s an overrides - screen shot 2017-01-03 at 1 41 48 pm](https://cloud.githubusercontent.com/assets/1217010/21599047/7d766442-d1ba-11e6-8b39-87c6f0df3852.png)